Cargo.toml

```diff
@@ -49,7 +49,7 @@ critical-section = [
 disable-cache-padding = [
     "maitake-sync?/no-cache-pad",
 ]
-std = []
```

Comment:
Could you please also add (or keep) the `std` feature, and just have it activate `alloc`? It doesn't matter now, but would be nice if we ever want it later.